README.md

```diff
@@ -45,21 +56,23 @@ BOOM runs on macOS and Linux. You'll need:
     source .venv/bin/activate
     uv pip install -r requirements.txt
     ```
-2. Next, copy the default config file, `config.default.yaml`, to `config.yaml`:
+    Note: If requirements.txt is missing, it can be skipped. It's only required for the ML pipeline.
```

Comment:
Maybe steps 1 and 2 should simply be deleted in light of #113.

Reply:
Ah I think you could have just added it to this PR, but that's fine. Let's merge this one first and then the other.